pom.xml

```diff
@@ -1,27 +1,28 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
         <version>4.33</version>
-        <relativePath />
+        <relativePath/>
     </parent>
     <groupId>com.atlassian.jira.software.cloud.jenkins</groupId>
     <artifactId>atlassian-jira-software-cloud</artifactId>
     <version>2.0.0-SNAPSHOT</version>
     <packaging>hpi</packaging>
     <properties>
         <!-- Baseline Jenkins version you use to build the plugin. Users must have this version or newer to run. -->
-       <jenkins.version>2.249.3</jenkins.version>
-       <jenkins.baseline>2.249</jenkins.baseline>
+        <jenkins.version>2.249.3</jenkins.version>
+        <jenkins.baseline>2.249</jenkins.baseline>
         <jenkins-test-harness.version>2.49</jenkins-test-harness.version>
         <java.level>8</java.level>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <filterModified>true</filterModified>
         <mockito.version>2.27.0</mockito.version>
-        <jackson.version>2.12.1</jackson.version>
-        <jackson-databind.version>2.12.3</jackson-databind.version>
+        <jackson.version>2.13.1</jackson.version>
+        <jackson-databind.version>2.13.1</jackson-databind.version>
```

Comment:
Sorry, I hit "reformat code" on this file. The only change is the Jackson version numbers above and adding the JWT library further down. I had to do that because Maven complained about incompatible versions after I added the JWT dependency.